runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -58,6 +60,8 @@ local Range = require('vim.treesitter._range')
 ---| 'on_child_added'
 ---| 'on_child_removed'
 
+---@alias TSTrees table<integer, TSTree>
```

Comment:
I'd prefer if we remove this alias if that's ok?

We need to rename all the ts types any so they have a `vim.treesitter.*` prefix.